lib/mbaasRequest/mbaasRequest.js

```diff
@@ -173,6 +173,15 @@ function responseHandler(params, cb) {
       body: body
     });
 
+    // MbaaS is unreachable
+    if (err && err.code !== "undefined" && err.code !== null && err.code === "ENOTFOUND" || err.code === "ETIMEDOUT") {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1067)